reset_security_server.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Script should be run as sudo
+# Usage reset_security_server.sh -n=server(s) -h=path/to/hosts.txt -a=path/to/ansible
+# Usage example: reset_security_server.sh -n=ss3,ss4 -h=../../X-Road/ansible/hosts/lxd_hosts.txt -a=../../X-Road/ansible
```

Comment:
In general I agree, that for somebody not much familiar with the X-Road and related installation procedures, this may seem a bit overwhelming, but I think when somebody wishes to use the Toolkit, he or she should be familiar with the X-Road, namely how it works in general and how to install and set it up, because the Toolkit is not an independent tool.

I will add a couple of lines for description, but I am in the opinion, that the Toolkit's documentation is not the place to start describing the X-Road - what it is and how it can be installed. An option would be to add a small chapter in its documentation, where some relevant places from the X-Road documentation are referred by hyperlinks.

Reply:
Hey, don't go too fast here, there might be a misunderstanding :) I did not mean for you to describe **X-Road** and what it is. But the file you are referring to comes from **X-Road** source tree (from parts which are not in binary distribution packages at least, that administrators now install). So what I meant was that end-user needs reference to that project (https://github.com/nordic-institute/X-Road) from where you are taking the files, not about the full **X-Road** concept.

In one of my previous comments I also mused that this dependency maybe could be stabilized (or extracted) and version match between toolkit / X-Road defined.

Reply:
Amended the description in the file header and referred to X-Road repository.